.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ jobs:
         env:
           POSTGRES_USER: mds
           POSTGRES_PASSWORD: mds
-          POSTGRES_DB: mds
```

Comment:
were you running into some issue with the default name?

Reply:
For my local development environment, I had a `mds` Postgres database set up and `DB_DATABASE=mds` in `.env`.  Because the `if TESTING:` block came after `DB_DSN = config(...)` in `src/mds/config.py`, running `pytest` locally also connects to the `mds` database, causing the `metadata` table to be dropped and the records I had created from Postman to be destroyed.

Obviously one way to get around this would be to just set up another database locally and configure `DB_DATABASE` appropriately (e.g. `DB_DATABASE=dev_mds`), but then every time you switch between running `pytest` vs `python run.py`, you have to make sure that `DB_DATABASE` is set correctly.

Since `environ["TESTING"] = "TRUE"` at the top of `tests/conftest.py` and we already had `DB_DATABASE = "test_" + (DB_DATABASE or "mds")` in the `if TESTING:` block in the config file, I thought it would be simplest to use `mds` for the local development database and `test_mds` for the automated tests (both locally and in Travis).